payment/models.py

```diff
@@ -58,6 +60,53 @@ class PaymentRequest(ModelWithStr):
     callback = StringType()  # a webhook to call when a payment is complete
 
 
+class WhitelistRequest(ModelWithStr):
+    order_id = StringType()
+    source = StringType()
+    destination = StringType()
+    amount = IntType()
+    xdr = StringType()
+    network_id = StringType()
+    app_id = StringType()
+
+    @staticmethod
+    def compare_attr(attr1, attr2, attr_name):
+        if attr1 != attr2:
+            raise TransactionMismatch('{attr_name}: {attr1} does not match expected {attr_name}: {attr2}'.
```

Comment:
aren't you using format strings with `f` prefix?

Reply:
Im do it with new projects, but this project is running on Python 3.5 and I dont want to change it now